site/assets/base.css

```diff
@@ -119,3 +119,9 @@ a {
     opacity: 1;
   }
 }
+
+@media not all and (min-resolution: 0.001dpcm) {
+  img[loading='lazy'] {
+    clip-path: inset(0.5px);
```

Comment:
Could you expand on this?

Reply:
This I took from the [docs](https://nextjs.org/docs/api-reference/next/image#known-browser-bugs), can you explain more ?